README.md

```diff
@@ -152,27 +152,26 @@ From Helm chart versions `2.2.0` and above, update your Pega Platform version to
 
 # Debugging failed upgrades using helm commands
 
-Sometimes, upgrades using helm chart might fail in the middle due to invalid configurations, networking issue or some platform issue. The first step during the failure event
-should be to do perform basic troubleshooting which is to collect the logs and check for any known issue.
+Upgrades using helm charts may fail in due to a variety issues, including an invalid configuration, a networking issue, or a platform issue. To diagnose the issue or issues to begin troubleshooting, begin by reviewing failure events in the logs and check for a detailed error; after understanding its cause, you can begin troubleshooting the issue or issues.
```

Comment:
sorry I didn't give you a really clear, clean edit. pleas tweak this. again, I'm sorry>>
Upgrades using helm charts may fail due to a variety issues, including an invalid configuration, a networking issue, or a platform issue. To diagnose the issue or issues, review failure events in the logs and check for a detailed error; after understanding its cause, you can begin troubleshooting the issue or issues.